docs/getting-started/locally.md

```diff
@@ -0,0 +1,104 @@
+# nio Local Quickstart
+
+The cloud is an easy way to get the nio Platform up and running, but to see the power of nio as a distributed system, you should install and run nio on a locally or at the edge.
+
+>**[info] Requirements**
+>
+>* **Python3**: https://www.python.org/downloads (you need Python 3.4+)
+>* **Git**: https://git-scm.com/downloads (nio uses git to manage projects)
+>* **nio**: https://app.n.io/binaries/download (you'll need to agree to the license)
+>  * Windows users should download the Windows Installer. Double-click and follow the prompts.
+
+## Installation
```

Comment:
Should add a section for downloading the wheel or at least specify that you will need a wheel file. In other words, expand on the "nio" requirement above.